### PR TITLE
COP-3430 : Add E2E tests for Password reset functionality

### PR DESCRIPTION
**Pull request by Bothees**
## Description
This PR has e2e tests for password reset functionality. 

### How to run the tests ?
npm run cypress:test:local -- --env mailSlurpApiKey=ApiKey --spec cypress/integration/sGMR/password-reset.spec.js

## Developer Checklist

\* Required

- [x] Up-to-date with master branch? \*

- [x] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [x] All reviewer comments resolved/answered? \*

Ensure you delete the branch once the Pull Request is merged.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
         "cors": "^2.8.5",
         "govuk-frontend": "^3.5.0",
         "history": "^4.10.1",
-        "lodash.debounce": "^4.0.8",
+        "mailslurp-client": "^8.9.1",
```
Comment (rafenden) [inline package.json:97]: Please move this package to dev dependencies.

File: cypress/support/commands.js [hunk 2]
```diff
@@ -279,3 +279,11 @@ Cypress.Commands.add('activateAccount', () => {
     });
   });
 });
+
+Cypress.Commands.add('resetPassword', () => {
```
Comment (rafenden) [inline cypress/support/commands.js:283]: Please remove this commands, because it's never used.

**Review by rafenden — CHANGES_REQUESTED** [review]

**Review by ajames14 — APPROVED** [review]

File: package-lock.json [hunk 3]
```diff
@@ -11203,8 +11203,7 @@
     "qs": {
       "version": "6.9.4",
       "resolved": "https://registry.npmjs.org/qs/-/qs-6.9.4.tgz",
-      "integrity": "sha512-A1kFqHekCTM7cz0udomYUoYNWjBebHm/5wzU/XqrBRBNWectVH0QIiN+NEcZ0Dte5hvzHwbr8+XQmguPhJ6WdQ==",
-      "dev": true
```
Comment (rafenden) [inline package-lock.json:11207]: Did you change `package-lock.json` manually or it was generated?

Comment (Bothees) [reply package-lock.json:11207]: I ran npm install & it was generated.

**Review by rafenden — APPROVED** [review]

**Pull request merged**